sherlock.py

```diff
@@ -464,6 +467,14 @@ def main():
                 f"Error: Desired sites not found: {', '.join(site_missing)}.")
             sys.exit(1)
 
+    if args.rank:
+        # Sort data by rank 
+        site_dataCpy = dict(site_data)
+        ranked_site_data = sorted(site_data, key=lambda k: site_data[k]['rank'])
```

Comment:
It may error if `rank` key doesn't exists for some sites; I think writing it this would be better: 
`ranked_site_data = sorted(site_data, key=lambda k: ("rank" not in k, site_data[k].get("rank", 0)))`

Reply:
Thanks, it's done! I just used `sys.maxsize` instead of `0` for the default values, so that websites whose rank we don't know don't get shown at the top but in the bottom.

Reply:
Yes, It's better 👍